maintenance_plan/views/maintenance_equipment_views.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- equipment : views inheritance -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="name">equipment.form.inherit</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="model">maintenance.equipment</field>
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <button name="%(maintenance_plan_equipment_action)d"
+                        type="action"
+                        class="oe_stat_button"
+                        icon="fa-list">
+                    <field string="Plans" name="maintenance_plan_count" widget="statinfo"/>
+                </button>
+            </div>
+            <xpath expr="//group[@name='maintenance']" position="replace">
```

Comment:
I think it is better to replace the group above it so the one2many list can be expanded to the whole scren.
//group[@name='maintenance']/..

![imagen](https://user-images.githubusercontent.com/47532076/68289251-17045900-0086-11ea-88b5-be3b546e995d.png)

Reply:
I did not know about this notation. Thanks! :+1: